index.html

```diff
@@ -281,6 +286,10 @@ <h3>Audio stream with titles and images</h3>
           messages to be displayed, with information about when the content
           should be displayed in relation to the <a>media timeline</a>.
         </p>
+        <p>
+          The visual information should be rendered within a hundred millseconds
```

Comment:
s/millseconds/milliseconds

Reply:
Thank you! I have run the document through a spell checker, which picked up a few more mistakes.